lib/pages/home_page.dart

```diff
@@ -0,0 +1,96 @@
+import 'dart:async';
+import 'package:deskify/main.dart';
+import 'package:deskify/provider/desk_provider.dart';
+import 'package:deskify/provider/profile_provider.dart';
+import 'package:deskify/widgets/tabs/add_device_tab.dart';
+import 'package:deskify/widgets/tabs/home_page_tab.dart';
+import 'package:deskify/widgets/tabs/settings_tab.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class HomePage extends StatefulWidget {
+  const HomePage({super.key});
+
+  @override
+  State<HomePage> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  final double standingBreakpointHeight = 90.0;
+
+  int selectedTabIndex = 0;
+  DeskProvider? deskProvider;
+  ProfileProvider? profileProvider;
+  late Timer analyticSecondTimer;
```

Comment:
search for places where I can use the `late`-keyword

Reply:
propably for the providers everytime -> I can save me the `!` everytime I try to use the provider